tests/integration/test_containerd_integration.py

```diff
@@ -1,25 +1,32 @@
 import logging
 
 import pytest
+import toml
 
 log = logging.getLogger(__name__)
 
 
 @pytest.mark.abort_on_fail
 async def test_build_and_deploy(ops_test):
     """Build and deploy Containerd in bundle."""
-    containerd_charm = await ops_test.build_charm(".")
     bundle = ops_test.render_bundle(
-        "tests/data/bundle.yaml", master_charm=containerd_charm, series="focal",
+        "tests/data/bundle.yaml",
+        master_charm=await ops_test.build_charm("."),
```

Comment:
Why was `series="focal"` removed? I still see `{{ series }}` in the bundle template.

Reply:
no idea -- reverting